lighthouse-core/scripts/lantern/collect/common.js

```diff
@@ -20,12 +20,17 @@ const collectFolder = `${LH_ROOT}/dist/collect-lantern-traces`;
 const summaryPath = `${collectFolder}/summary.json`;
 const goldenFolder = `${LH_ROOT}/dist/golden-lantern-traces`;
 
+const IS_INTERACTIVE = !!process.stdout.isTTY && !process.env.GCP_COLLECT;
+
 class ProgressLogger {
   constructor() {
     this._currentProgressMessage = '';
     this._loadingChars = '⣾⣽⣻⢿⡿⣟⣯⣷ ⠁⠂⠄⡀⢀⠠⠐⠈';
     this._nextLoadingIndex = 0;
-    this._progressBarHandle = setInterval(() => this.progress(this._currentProgressMessage), 100);
+    this._progressBarHandle = setInterval(
+      () => this.progress(this._currentProgressMessage),
+      IS_INTERACTIVE ? 100 : 5000
```

Comment:
:)

was this progress indicator to being cleared in w/e output viewer for GCP you were looking at?

Reply:
it was flooding the disk with log updates every 100 ms :) having some lines printed to ensure it's still working is nice, but just less frequent 👍